charts/fluentd/templates/_pod.tpl

```diff
@@ -33,23 +33,25 @@ containers:
       {{- end }}
       exec /fluentd/entrypoint.sh
   {{- end }}
-  {{- if .Values.env }}
     env:
+      - name: FLUENTD_CONF
+        value: "../../../etc/fluent/fluent.conf"
+    {{- if .Values.env }}
     {{- toYaml .Values.env | nindent 6 }}
-  {{- end }}
+    {{- end }}
   {{- if .Values.envFrom }}
     envFrom:
     {{- toYaml .Values.envFrom | nindent 6 }}
   {{- end }}
     ports:
-    - name: metrics
-      containerPort: 24231
-      protocol: TCP
-    {{- range $port := .Values.service.ports }}
-    - name: {{ $port.name }}
-      containerPort: {{ $port.containerPort }}
-      protocol: {{ $port.protocol }}
-    {{- end }}
+      - name: metrics
```

Comment:
I am not really a fan of this type of identing, just because of how out of fase the first parameter and the rest of the dict object stay out of fase with the `ports` key.

will still merge it, but please change? 🥺

Reply:
I'm not sure what type of indenting you're referring to here. Do you want the metrics bumped to the left two spaces?

Reply:
I will validate all the identation changes you made in one go